eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -50,6 +50,16 @@
       "TestFromSource": false,
       "TestGoals": "surefire:test",
       "TestOptions": ""
+    },
```

Comment:
I'll have to look, but I wonder if there are some opportunities to consolidate the matrix again.